doc/developers.rst

```diff
@@ -83,7 +83,7 @@ Move to the project's root directory and execute the following command:
 
 .. code-block:: shell
 
-   $ python -m unittest discover tests
+   $ python -m unittest discover -b tests
```

Comment:
The ``-b`` switch suppresses the program output (like *Server up and running*), leaving just the tests result